hcp_vault/assets/dashboards/hcp_vault_overview.json

```diff
@@ -75,7 +74,7 @@
                                     "response_format": "scalar",
                                     "queries": [
                                         {
-                                            "query": "avg:hcp.vault_core_unsealed{$cluster}",
+                                            "query": "sum:hcp.vault_core_unsealed{$cluster,$project_id}",
```

Comment:
silent change: can we pls document this in the PR description as well?

Reply:
Ah yes. I think this looked as a pre-existing bug for me which we cleared out now.